Sources/InteroperabilityTestsExecutable/InteroperabilityTestsExecutable.swift

```diff
@@ -0,0 +1,64 @@
+import ArgumentParser
+import GRPCCore
+import GRPCHTTP2Core
+import GRPCHTTP2TransportNIOPosix
+import NIOPosix
+
+@testable import InteroperabilityTests
```

Comment:
Hmm we shouldn't need `@testable` here, the `InteroperabilityTests` module won't ever be a product so any API you need from it can be `public`

Reply:
Oh okay. I forgot about this, I was going to suggest having an SPI, but you're right it can simply be public.